conference.js

```diff
@@ -1548,7 +1550,7 @@ export default {
             // again switching to the screen sharing.
             APP.UI.showExtensionInlineInstallationDialog(
                 () => {
-                    this.toggleScreenSharing();
+                    this.toggleScreenSharing().catch(() => {});
```

Comment:
Because of this comment: https://github.com/jitsi/jitsi-meet/pull/1911/commits/0ae793de4c15eb1ff5dc6e4a47ea70c33dca195d#diff-f1dfd122026e13422664665b42231196L1519
It is important to test this case